### PR TITLE
Rename contracts and variables

### LINKED ISSUE #208: Rename contracts and variables
Rename
- [ ] `newMetaChain` to `newMetachain`
- [ ] `CoConsensusModule` to `CoconsensusModule`
- [ ] `CoConsensusI` to `CoconsensusInterface`
- [ ] `setupCoConsensus` to `setupCoconsensus`
- [ ] `consensusCoGateway` to `consensusCogateway`

- [ ] Interfaces should have the `Interface` postfix in naming.
  `Example`
  > ConsensusInterface, GatewayInterface

### PR DESCRIPTION
**Pull request by kreloaded**
Fixes #208 

Checked for following contracts and variables whether they are already changes or not:
- newMetaChain to newMetachain
- CoConsensusModule to CoconsensusModule
- CoConsensusI to CoconsensusInterface
- setupCoConsensus to setupCoconsensus
- consensusCoGateway to consensusCogateway

Required major changes in interface names:
Changes in following interfaces names are done.
- CoconsensusI => CoconsensusInterface
- AxiomI => AxiomInterface
- ObserverI => ObserverInterface
- CommitteeI => CommitteeInterface
- ConsensusI => ConsensusInterface
- ConsensusGatewayI => ConsensusGatewayInterface
- ConsensusCogatewayI => ConsensusCogatewayInterface
- CoreI => CoreInterface
- StateRootI => StateRootInterface
- MOSTI => MOSTInterface
- ProtocoreI => ProtocoreInterface
- SelfProtocoreI => SelfProtocoreInterface
- CoreputationI => CoreputationInterface
- ReputationI => ReputationInterface

### CODE REVIEW

File: contracts/message-bus/MessageInbox.sol [hunk 1]
```diff
@@ -15,7 +15,7 @@ pragma solidity >=0.5.0 <0.6.0;
 // limitations under the License.
 
 import "./MessageBox.sol";
-import "./StateRootI.sol";
+import "./StateRootInterface.sol";
```
Comment (gulshanvasnani) [inline contracts/message-bus/MessageInbox.sol:18]: As we have touched this section, i think we must sort the imports.

File: contracts/test/consensus/MockConsensus.sol [hunk 2]
```diff
@@ -54,12 +54,12 @@ contract MockConsensus is ConsensusI, ReputationI {
 
         mockCore = new MockCore();
         mockCore.setup(
-			ConsensusI(address(this)),
+			ConsensusInterface(address(this)),
```
Comment (gulshanvasnani) [inline contracts/test/consensus/MockConsensus.sol]: It is misaligned.

**Review by gulshanvasnani — COMMENTED** [review]
Couple of inline comments.
I think we can sort the imports.

File: contracts/message-bus/MessageInbox.sol (hunk 1 above)
Comment (kreloaded) [reply contracts/message-bus/MessageInbox.sol:18]: Done.

File: contracts/consensus-gateway/ConsensusGateway.sol [hunk 3]
```diff
@@ -14,17 +14,17 @@ pragma solidity >=0.5.0 <0.6.0;
...
 import "./ERC20GatewayBase.sol";
-import "../proxies/MasterCopyNonUpgradable.sol";
-import "../message-bus/MessageBus.sol";
-import "../consensus/ConsensusModule.sol";
 import "../consensus/ConsensusInterface.sol";
+import "../consensus/ConsensusModule.sol";
 import "../core/CoreInterface.sol";
+import "../proxies/MasterCopyNonUpgradable.sol";
+import "../message-bus/MessageBus.sol";
 import "./ConsensusGatewayInterface.sol";
```
Comment (gulshanvasnani) [inline contracts/consensus-gateway/ConsensusGateway.sol]: It would be come at line 18.

File: contracts/reputation/Reputation.sol [hunk 4]
```diff
@@ -14,12 +14,12 @@ pragma solidity >=0.5.0 <0.6.0;
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import "openzeppelin-solidity/contracts/math/SafeMath.sol";
-
-import "../ERC20I.sol";
 import "../consensus/ConsensusModule.sol";
+import "../ERC20I.sol";
```
Comment (gulshanvasnani) [inline contracts/reputation/Reputation.sol:17]: It would come before line 17.

**Review by gulshanvasnani — COMMENTED** [review]
Almost done.
Couple of inline comments.

File: contracts/message-bus/Proof.sol [hunk 5]
```diff
@@ -14,11 +14,11 @@ pragma solidity >=0.5.0 <0.6.0;
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import "./StateRootInterface.sol";
+import "../lib/BytesLib.sol";
 import "../lib/CircularBufferUint.sol";
-import "./StateRootI.sol";
 import "../lib/RLP.sol";
 import "../lib/MerklePatriciaProof.sol";
```
Comment (gulshanvasnani) [inline contracts/message-bus/Proof.sol:21]: It would come before `import "../lib/RLP.sol";`

File: contracts/consensus/Consensus.sol [hunk 6]
```diff
@@ -14,21 +14,21 @@ pragma solidity >=0.5.0 <0.6.0;
...
 
-import "openzeppelin-solidity/contracts/math/SafeMath.sol";
-
-import "./ConsensusI.sol";
+import "./ConsensusInterface.sol";
 import "./CoreLifetime.sol";
-import "../anchor/AnchorI.sol";
-import "../axiom/AxiomI.sol";
+import "../anchor/AnchorInterface.sol";
+import "../axiom/AxiomInterface.sol";
```
Comment (gulshanvasnani) [inline contracts/consensus/Consensus.sol:20]: It would come before line 17.

File: contracts/consensus-gateway/ConsensusGateway.sol (hunk 3 above)
Comment (kreloaded) [reply contracts/consensus-gateway/ConsensusGateway.sol]: Done.

File: contracts/reputation/Reputation.sol (hunk 4 above)
Comment (kreloaded) [reply contracts/reputation/Reputation.sol:17]: Done.

**Review by gulshanvasnani — APPROVED** [review]
LGTM 👍

**Review by deepesh-kn — APPROVED** [review]

**Pull request merged**